clusterDeployer.py

```diff
@@ -384,7 +386,11 @@ def create_workers(self) -> None:
         # provisioning node.
         for h in hosts_with_workers:
             h.configure_bridge()
-            h.configure_worker_dhcp_entries(self._local_host.bridge)
+        node_configs: list[NodeConfig] = [node.config for h in hosts_with_workers for node in h.k8s_worker_nodes if node.config.kind == "vm"]
```

Comment:
nested for loop in 1 statement. We might want to break this apart, as it is difficult to read.

Reply:
Move to clusterConfig as well.